elkscmd/cron/cron.c

```diff
@@ -225,7 +225,7 @@ printcrontab(crontab *tab, int nrtab)
                 for (j = 0; j < tabs[i].nrl; j++) {
                     printf("    ");
                     printtrig(&tabs[i].list[j].trigger);
-                    printf("%s", tabs[i].list[j].command);
+                    fputs(tabs[i].list[j].command, stdout);
```

Comment:
Are you sure that replacing `printf` with `fputs` is actually smaller here? In the case where `printf` is already used, it seems this just drags in another function, rather than saving space as it would if `printf` could actually be replaced in full at link time. Same for a few other cases with printf->fputs substitution in this commit.

Reply:
Right, changing back to printf saves 16 bytes on the binary.  Done.